test/math.jl

```diff
@@ -387,12 +387,20 @@ j33 = besselj(3,3.)
 @test besselj(-3,-3) == j33
 @test besselj(-3,3) == -j33
 @test besselj(3,-3) == -j33
+@test besselj(3,3f0) ≈ j33
```

Comment:
do these have to be approximate because we can't reliably constrain different libm's accuracy here?